src/plugins/timeline/timelineInterceptor.js

```diff
@@ -20,18 +20,24 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
+import { configuration } from './configuration.js';
+
 export default function timelineInterceptor(openmct) {
   openmct.objects.addGetInterceptor({
     appliesTo: (identifier, domainObject) => {
       return domainObject && domainObject.type === 'time-strip';
     },
     invoke: (identifier, object) => {
       if (object && object.configuration === undefined) {
-        object.configuration = {
-          useIndependentTime: true
-        };
+        object.configuration = configuration;
```

Comment:
See other comment about webpack caching.

Reply:
Good catch. Corrected this issue.